src/index.ts

```diff
@@ -16,6 +17,7 @@ export interface CoreOptions {
   proxy?: string;
   multipart?: RequestPart[];
   forever?: boolean;
+  agent?: Agent;
```

Comment:
So is this something a user can pass into the options then?

Reply:
Yes! But it's needed for this feature, so I've removed it.